homeassistant/components/isy994/number.py

```diff
@@ -95,14 +119,17 @@ async def async_setup_entry(
         )
 
     for node, control in isy_data.aux_properties[Platform.NUMBER]:
+        entity_init_info = {
+            "node": node,
+            "control": control,
+            "unique_id": f"{isy_data.uid_base(node)}_{control}",
+            "description": CONTROL_DESC[control],
+            "device_info": device_info.get(node.primary_node),
+        }
         entities.append(
-            ISYAuxControlNumberEntity(
-                node=node,
-                control=control,
-                unique_id=f"{isy_data.uid_base(node)}_{control}",
-                description=CONTROL_DESC[control],
-                device_info=device_info.get(node.primary_node),
-            )
+            ISYBacklightNumberEntity(**entity_init_info)
+            if control == CMD_BACKLIGHT
+            else ISYAuxControlNumberEntity(**entity_init_info)
```

Comment:
Please unpack the ternary into an if block when it spans multiple lines

Reply:
Will get this when I get back to a computer